src/main/scala/com/payalabs/scalajs/react/bridge/ReactBridgeComponent.scala

```diff
@@ -123,3 +125,11 @@ object ReactBridgeComponent {
     case value => value.asInstanceOf[js.Any]
   }
 }
+
+/**
+ *  extends this, to convert scala classes to js literals
+ */
+trait ReactBridgeObject extends Any {
+  def toJS : js.Dynamic
```

Comment:
yeah first i had same type , but tempted to reuse toLiteral , what u suggest foor new mwthod converter toJSValue ?